openslides/core/static/templates/projector-container.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
-<html lang="en" class="no-js">
+<html ng-controller="ProjectorContainerCtrl" lang="en" class="no-js">
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <base href="/">
-<title>OpenSlides &ndash; Projector</title>
+<title>OpenSlides &ndash; Projector {{ projectorId }}</title>
```

Comment:
@emanuelschuetze there is no languageCtrl on projector site (compare index.html to projector-container.html), so I'm not shure how to manage this. But what determinates the language for the projector? (I suggest the browser language, right?)